scripts/qa_crowbarsetup.sh

```diff
@@ -3491,6 +3495,15 @@ function oncontroller_run_tempest()
     fi
     testr last --subunit | subunit-1to2 > tempest.subunit.log
 
+    if [ -n "$ostestroptions" ]; then
+        zypper -n in python-os-testr
+        ostestr $ostestroptions 2>&1 | tee ostestr.log
```

Comment:
I can't evaluate how important is for QA have tests for magnum. But sure, lets unlock our CI first.